src/components/Button.tsx

```diff
@@ -93,15 +94,15 @@ export const BackButtonRouterIndexes: FunctionComponent<BackButtonProps> = ({
 export const LinkButton: FunctionComponent<LinkButtonProps> = ({
   buttonText,
   href,
+  classes,
 }: LinkButtonProps): JSX.Element => (
-  <Link href={href}>
-    <a
-      role="button"
-      draggable="false"
-      className="govuk-button"
-      data-module="govuk-button"
-    >
-      {buttonText}
-    </a>
-  </Link>
+  <a
+    href={href}
+    role="button"
+    draggable="false"
+    className={"govuk-button " + classes}
+    data-module="govuk-button"
+  >
+    {buttonText}
+  </a>
```

Comment:
The `<Link>` component was causing issues with Cypress detecting a link with the ARIA role `button`.  I looked into the Link component and it turns out it's designed for [client-side routing](https://nextjs.org/docs/api-reference/next/link).  Replacing with a good old fashioned HTML href to make Cypress happy, and also possibly make our site more accessible :-)